content/docs/docker_demo.html

```diff
@@ -543,7 +543,7 @@ <h3 id="step-2-incrementally-ingest-data-from-kafka-topic">Step 2: Incrementally
 <p>You can use HDFS web-browser to look at the tables
 <code class="highlighter-rouge">http://namenode:50070/explorer.html#/user/hive/warehouse/stock_ticks_cow</code>.</p>
 
-<p>You can explore the new partition folder created in the table along with a “deltacommit”
+<p>You can explore the new partition folder created in the table along with a “commit”/“deltacommit”
```

Comment:
@vikrantgoel can you please change the .md file? html is generated off that

Reply:
@lamber-ken @vinothchandar Thanks, updated.